packages/app-check/src/definitions.ts

```diff
@@ -93,6 +93,20 @@ export interface InitializeOptions {
    * @default false
    */
   debug?: boolean;
+  /**
+   * If debug token is set `true`, the predefined debug token can be used. If not set, a random debug token will be generated by Firebase SDK.
+   *
+   * Only available for Web. iOS and Android will use environment variables for this.
+   * 
+   * Read more:
+   * - [Web](https://firebase.google.com/docs/app-check/web/debug-provider)
+   * - [iOS](https://firebase.google.com/docs/app-check/ios/debug-provider#ci)
+   * - [Android](https://firebase.google.com/docs/app-check/android/debug-provider#ci)
+   *
+   * @since TBA
+   * @default false
+   */
+  debugToken?: string;
```

Comment:
```suggestion
  /**
   * If `true`, the debug provider is used.
   *
   * On **Web**, you can also set a predefined debug token string instead of `true`. On Android and iOS, you have to use environment variables for this.
   *
   * @since 7.1.0
   * @default false
   * @see https://firebase.google.com/docs/app-check/android/debug-provider#ci
   * @see https://firebase.google.com/docs/app-check/ios/debug-provider#ci
   * @see https://firebase.google.com/docs/app-check/web/debug-provider
   */
  debugToken?: boolean | string;
```

Let's deprecate the `debug` property and use this property for all platforms.

Reply:
Good idea, I can change the logic.